--- textflow-0.2.9/textflow.desktop	2010-12-26 18:16:45.000000000 -0600
+++ textflow-0.2.9/textflow.desktop.old	2010-12-26 18:15:18.000000000 -0600
@@ -1,9 +1,9 @@
 [Desktop Entry]
 Name=TextFlow
 #_GenericName=Package Backup Tool
-GenericName=Text Editor
-Comment=Programmer's editor
-Version=1.0
+_Comment=Text Editor for Programming
+Version=0.2
+Encoding=UTF-8
 Icon=textflow
 Exec=textflow %U
 Terminal=false
